src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/CSharpCodeRefactoringVerifier`1+Test.cs

```diff
@@ -84,9 +91,70 @@ public Test()
 
             public string? EditorConfig { get; set; }
 
+            /// <summary>
+            /// The set of code action <see cref="CodeAction.Title"/>s offered the user in this exact order.
+            /// Set this to ensure that a very specific set of actions is offered (or not).
+            /// </summary>
+            public string[]? ExactActionSetOffered { get; set; }
+
+            protected override ImmutableArray<CodeAction> FilterCodeActions(ImmutableArray<CodeAction> actions)
+            {
+                var result = base.FilterCodeActions(actions);
+
+                if (ExactActionSetOffered != null)
+                {
+                    Verify.Equal(ExactActionSetOffered.Length, result.Length, "Different number of code actions provided");
+                    Verify.SequenceEqual(ExactActionSetOffered, result.SelectAsArray(a => a.Title));
```

Comment:
The error message in this case won't be helpful. Only `SequenceEqual` is needed.

```suggestion
                    Verify.SequenceEqual(ExactActionSetOffered, result.SelectAsArray(a => a.Title));
```